notebooks/tm/landcover/01-extract-landcover-features.ipynb

```diff
@@ -0,0 +1,1084 @@
+{
```

Comment:
<p>Delete header and move function code cell below under the next header</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>